File: mobile/lib/models/crop.dart
class Crop {
  final String id;
  final String cropName;
  final String variety;
  final int optimalGrowthDurationDays;
  final String? description;

  Crop({
    required this.id,
    required this.cropName,
    required this.variety,
    required this.optimalGrowthDurationDays,
    this.description,
  });

  factory Crop.fromJson(Map<String, dynamic> json) {
    return Crop(
      id: json['id'],
      cropName: json['cropName'],
      variety: json['variety'],
      optimalGrowthDurationDays: json['optimalGrowthDurationDays'],
      description: json['description'],
    );
  }
}

File: backend/AgriOpsAI.Api/Models/Crop.cs
namespace AgriOpsAI.Api.Models;

public class Crop
{
    public Guid Id{get;set;}
    public string CropName {get;set;} = string.Empty;
    public string Variety {get; set;} = string.Empty;
    public int OptimalGrowthDurationDays {get; set;}
    public string? Description {get;set;}

    public ICollection<CropSeason> CropSeasons {get;set;} = new List<CropSeason>();

}